src/App.jsx

```diff
@@ -14,70 +16,73 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
+  return (
+    <main className="section container">
+      {selectedGood
+        ? (
+          <h1 className="title is-flex is-align-items-center">
+            {`${selectedGood} is selected`}
             <button
-              data-cy="AddButton"
+              data-cy="ClearButton"
               type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+              className="delete ml-3"
+              onClick={() => setSelectedGood('')}
+            />
+          </h1>
+        )
+        : (
+          <h1 className="title is-flex is-align-items-center">
+            No goods selected
+          </h1>
+        )
+      }
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
+      <table className="table">
+        <tbody>
+          {goods.map(good => (
+            <tr
```

Comment:
try not repeat yourself
move good === selectedGood to a variable
```suggestion
          {goods.map(good => {
            const isActiveGood = good === selectedGood;
            
            return (
              <tr
```